.gitlab-ci.yml

```diff
@@ -562,7 +562,11 @@ library:ci-argosy:
   extends: .ci-template
 
 library:ci-bedrock2:
-  extends: .ci-template
+  extends: .ci-template-flambda
+  artifacts:
+    name: "$CI_JOB_NAME"
+    paths:
+      - _build_ci
```

Comment:
Should we maybe encourage using `_build_ci/bedrock2` etc... ? @SkySkimmer ?

Reply:
I'm fine with using something stricter if it works.